### PR TITLE
Simplify `imagePromises` in reftest-analyzer.js

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

File: webapp/components/reftest-analyzer.js [hunk 1]
```diff
@@ -180,16 +180,15 @@ class ReftestAnalyzer extends LoadingState(PolymerElement) {
 
     // Set the img srcs manually so that we can promisify them being loaded.
     const imagePromises = ['before', 'after'].map(prop => {
-      if (!this[prop]) {
-        return Promise.reject(`${prop} is empty`);
-      }
-      const img = this.shadowRoot.querySelector(`#${prop}`);
-      const loaded = new Promise((resolve, reject) => {
+      return new Promise((resolve, reject) => {
```
Comment (Hexcles) [inline webapp/components/reftest-analyzer.js]: Can we simplify this even further to get rid of the curly brackets and return?

Comment (foolip) [reply webapp/components/reftest-analyzer.js]: Yes, can do when at a computer.

File: webapp/components/reftest-analyzer.js [hunk 2]
```diff
@@ -179,17 +179,14 @@ class ReftestAnalyzer extends LoadingState(PolymerElement) {
...
       const img = this.shadowRoot.querySelector(`#${prop}`);
-      const loaded = new Promise((resolve, reject) => {
-        img.onload = resolve;
-        img.onerror = reject;
-      });
+      img.onload = resolve;
+      img.onerror = reject;
       img.src = this[prop];
-      return loaded;
     });
```
Comment (Hexcles) [inline webapp/components/reftest-analyzer.js]:
```suggestion
    }));
```

**Comment by Hexcles** [comment]
Force-merging this. The only failure is because of the branch name :)

**Pull request merged**